app-backup/zbackup/zbackup-9999.ebuild

```diff
@@ -18,8 +18,8 @@ IUSE="libressl tartool"
 DEPEND="app-arch/lzma
 	dev-libs/lzo:2
 	dev-libs/protobuf:0=
-	sys-libs/libunwind:0
-	sys-libs/zlib
+	sys-libs/libunwind:=
+	sys-libs/zlib:=
```

Comment:
please don't add the slot op in this case, it was added to zlib in error.

Reply:
Done